### PR TITLE
added unregister

### PR DESCRIPTION
**Pull request by MarkNorman**

### CODE REVIEW

File: Pod/Classes/CloudFivePush.m [hunk 1]
```diff
@@ -185,33 +193,53 @@ - (void)register:(NSString *)userIdentifier
...
+                          device.systemVersion,
+                          [bundle objectForInfoDictionaryKey:(NSString *)kCFBundleVersionKey],
+                          device.identifierForVendor.UUIDString];
 
-                          ];
     if (_uniqueIdentifier != nil) {
         postData = [postData stringByAppendingFormat:@"&user_identifier=%@", _uniqueIdentifier];
     }
 
+    NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"https://www.cloudfiveapp.com/push/register"]];
```
Comment (samsonasu) [inline Pod/Classes/CloudFivePush.m]: Should we change this to use push.cloudfiveapp.com going forward?

Comment (MarkNorman) [reply Pod/Classes/CloudFivePush.m]: I thought you hated that idea now?

**Comment by samsonasu** [comment]
Right now they are the same server, I think it gives us flexibility if it hits a different endpoint to move it, scale it, etc

File: Pod/Classes/CloudFivePush.m (hunk 1 above)
Comment (petitJAM) [reply Pod/Classes/CloudFivePush.m]: should i do this on the android library then?

**Comment by MarkNorman** [comment]
Let me know when the A record is setup and pointed and I will change the reference.

**Comment by samsonasu** [comment]
it's been set up forever, but you should test it to make sure it works


bsamson$ host push.cloudfiveapp.com

push.cloudfiveapp.com is an alias for www.cloudfiveapp.com.

**Comment by MarkNorman** [comment]
seems to work, but my device is registered under 2 GUIDs now.  i think that might be a different bug with a recent release.

**Pull request merged**